app/views/idv/doc_auth/_start_over_or_cancel.html.erb

```diff
@@ -1,9 +1,8 @@
-<br/>
 <%= button_to(
   t('doc_auth.buttons.start_over'),
   idv_doc_auth_step_path(step: :reset),
   method: :put,
   class: 'btn btn-link',
-  form_class: 'inline-block'
+  form_class: 'inline-block. mt3'
```

Comment:
I think the `.` here is unintended? With it, the `inline-block` styles are no longer applied:

```suggestion
  form_class: 'inline-block mt3'
```

Reply:
🤦‍♂️